set.mm

```diff
@@ -15746,103 +15746,148 @@ Equality predicate (continued)
-     can also use ` [ y / x ] ph ` in place of the "free for" side condition
-     used in traditional predicate calculus; see, for example, ~ stdpc4 .
-
-     Our notation was introduced in Haskell B. Curry's _Foundations of
-     Mathematical Logic_ (1977), p. 316 and is frequently used in textbooks of
-     lambda calculus and combinatory logic.  This notation improves the common
-     but ambiguous notation, " ` ph ( y ) ` is the wff that results when ` y `
-     is properly substituted for ` x ` in ` ph ( x ) ` ".  For example, if the
-     original ` ph ( x ) ` is ` x = y ` , then ` ph ( y ) ` is ` y = y ` , from
-     which we obtain that ` ph ( x ) ` is ` x = x ` .  So what exactly does
-     ` ph ( x ) ` mean?  Curry's notation solves this problem.
-
-     In most books, proper substitution has a somewhat complicated recursive
-     definition with multiple cases based on the occurrences of free and bound
-     variables in the wff.  Instead, we use a single formula that is exactly
-     equivalent and gives us a direct definition.  We later prove that our
-     definition has the properties we expect of proper substitution (see
-     theorems ~ sbequ , ~ sbcom2 and ~ sbid2v ).
-
-     Note that our definition is valid even when ` x ` and ` y ` are replaced
-     with the same variable, as ~ sbid shows.  We achieve this by having ` x `
-     free in the first conjunct and bound in the second.  We can also achieve
-     this by using a dummy variable, as the alternate definition ~ dfsb7 shows
-     (which some logicians may prefer because it does not mix free and bound
-     variables).  Another version that mixes free and bound variables is
-     ~ dfsb3 .  When ` x ` and ` y ` are distinct, we can express proper
-     substitution with the simpler expressions of ~ sb5 and ~ sb6 .
-
-     There are no restrictions on any of the variables, including what
-     variables may occur in wff ` ph ` .  (Contributed by NM, 10-May-1993.) $)
-  df-sb $a |- ( [ y / x ] ph <->
-              ( ( x = y -> ph ) /\ E. x ( x = y /\ ph ) ) ) $.
-
-  $( An equality theorem for substitution.  (Contributed by NM, 16-May-1993.)
-     (Proof shortened by Wolf Lammen, 25-Feb-2018.) $)
-  sbequ2 $p |- ( x = y -> ( [ y / x ] ph -> ph ) ) $=
-    ( wsb weq wi wa wex df-sb simplbi com12 ) ABCDZBCEZALMAFMAGBHABCIJK $.
-
-  $( One direction of a simplified definition of substitution.  The converse
-     requires either a disjoint variable condition ( ~ sb5 ) or a non-freeness
-     hypothesis ( ~ sb5f ).  (Contributed by NM, 13-May-1993.) $)
-  sb1 $p |- ( [ y / x ] ph -> E. x ( x = y /\ ph ) ) $=
-    ( wsb weq wi wa wex df-sb simprbi ) ABCDBCEZAFKAGBHABCIJ $.
+  ${
+    $d y x $.  $d y t $.  $d y ph $.
+    $( Define proper substitution.  For our notation, we use ` [ y / x ] ph `
+       to mean "the wff that results from the proper substitution of ` y ` for
+       ` x ` in the wff ` ph ` ".  That is, ` y ` properly replaces ` x ` .
```

Comment:
The statement defines ` [ t / x ] ` with dummy variable ` y ` while this part of the comment uses ` [ y / x ] ` . Either the comment or the statement has to be changed. I had used ` [ t / x ] ` and dummy ` y ` to underline that x, y are bound while t is free (t is like a "term"), but maybe it's better to follow the "mathematical alphabet": x, y, z...